README.md

```diff
@@ -1,21 +1,134 @@
+
+See [Deploying Code Remotely](#deploy-code) for info on how to deploy the code. 
+
+### Enabling Console
+
+This is not necessary, though receiving console output over a serial cable or Telnet / SSH connected to your PC gives insight into the application. Alternatively logs can be viewed in the DWS. You will need a serial cable and adapter for your computer, connected to the BrightSign's 3.5mm Serial Port.
+
+At the prompt, you can find the USB port by entering
+```bash
+ls /dev/tty*
+```
+
+For example, the USB port might be /dev/tty.usbserial in which case you can execute a command like,
+```
+screen /dev/tty.usbserial 115200 
+```
+
+Once connected, use the serial software of your choice (on linux and Mac, the "screen" command is quite popular and all you need).
+
+Power on the player with the SVC button pressed.  Be near your keyboard because in a few seconds you will see:
+
+```bash
+Automatic startup in 3 seconds, press Ctrl-C to interrupt.
+```
+
+Press Ctrl-C and you will see a prompt that says SECURE or BOLT (depending on the player model).  Type these commands:
+
+```bash
+console on
+reboot
+```
+
+## Development lifecycle
+---
+
+### Deploying Code to the Player
+
+### <a id="deploy-code"></a>
+
+The underlying shell scripts (used by `npm scripts`) require [jq](https://stedolan.github.io/jq/download/). 
+
+Working at the command line is made much easier through using environment variables, and that is simplified using [direnv](https://direnv.net/docs/installation.html).  Highly recommended.
+
+If the following environment variables have not been set, on your PC, go ahead and set them (direnv is your friend):
+
+```
+export PLAYER={your XC5 IP address or hostname}
+export PLAYER_PW={the pw for that player, by default it's serial number}
+```
+
+The file scripts/put uses the DWS API to move files to the player.  If your application builds additional files you will need to add them to that script in the line that lists the files to copy.  For example:
+
+```bash
+for f in dist/*.html dist/*.js src/autorun.brs
+```
+
+The current script copies all the html and js files in the dist folder, and the autorun from the source file.  If your app needs more files, add them to that line.
+
+To run the script you can either run it as a bash script from the command line, or from npm:
+
+```
+npm run cp
+```
+
+You should see the results of the webpack build, followed by a list of the files copied to the player.
+
+### Starting the App on the Player
+
+Just copying the files will not restart your application on the player.  From a serial or Telnet / SSH session to the player:
+
+Restarting the application can be done via, 
+* rebooting the player (`npm run rp`)
```

Comment:
We can also mention Reboot player using Control tab in DWS, if applicable.

Reply:
Yes - good looks, updated this bit.